packages/features/bookings/lib/handleBookingRequested.ts

```diff
@@ -0,0 +1,69 @@
+import { sendAttendeeRequestEmail, sendOrganizerRequestEmail } from "@calcom/emails";
+import { getWebhookPayloadForBooking } from "@calcom/features/bookings/lib/getWebhookPayloadForBooking";
+import getWebhooks from "@calcom/features/webhooks/lib/getWebhooks";
+import sendPayload from "@calcom/features/webhooks/lib/sendPayload";
+import logger from "@calcom/lib/logger";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+const log = logger.getChildLogger({ prefix: ["[handleConfirmation] book:user"] });
+
+/**
+ * Supposed to do whatever is needed when a booking is requested.
+ */
+export async function handleBookingRequested(args: {
+  evt: CalendarEvent;
+  booking: {
+    eventType: {
+      currency: string;
+      description: string | null;
+      id: number;
+      length: number;
+      price: number;
+      requiresConfirmation: boolean;
+      title: string;
+      teamId?: number | null;
+    } | null;
+    eventTypeId: number | null;
+    userId: number | null;
+    id: number;
+  };
+}) {
+  const { evt, booking } = args;
+
+  await sendOrganizerRequestEmail({ ...evt });
+  await sendAttendeeRequestEmail({ ...evt }, evt.attendees[0]);
```

Comment:
Can we create something that if anything fails, don't stop anything else? Then we can add failed to retry system.

Reply:
I am not sure how that would look. Maybe we can do that when we implement retry system?